gpkit/model.py

```diff
@@ -544,6 +544,30 @@ def _latex(self, unused=None):
                                  for var, val in self.constants.items()]) +
                          ["\\end{array}"])
 
+    def interact(self,  fn_of_sol=None, ranges=None, **solveargs):
+        try:
+            from ipywidgets import interactive, FloatSlider
+        except ImportError:
+            from IPython.html.widgets import interactive, FloatSliderWidget
+            FloatSlider = FloatSliderWidget
+
+        if ranges is None:
+            ranges = {k._cmpstr: FloatSlider(min=v/10.0, max=10*v,
+                                             step=v/10.0, value=v)
+                      for k, v in self.constants.items()}
+        if fn_of_sol is None:
+            def fn_of_sol(solution):
+                print solution.table(["cost", "freevariables"])
```

Comment:
I don't think most users would want sensitivities and no free variables.

With the modular models we could print only the highest-level layer, right? So that a too-long table would be an indication to the user that they might want to break up their model into pieces? This thought started as "maybe the user should tag the variables they want to see printed" -- but it seems like Modular models give us a clean way to do that already.